wordcount.py

```diff
@@ -39,16 +39,34 @@
 
 import sys
 
-# +++your code here+++
-# Define print_words(filename) and print_top(filename) functions.
-# You could write a helper utility function that reads a file
-# and builds and returns a word/count dict for it.
-# Then print_words() and print_top() can just call the utility function.
+def print_words(filename):
+    import operator
+    d = word_dict(filename)
```

Comment:
You should name your variables properly. Here `d` could be better named in `words`.